static/src/javascripts/projects/common/views/svgs.js

```diff
@@ -63,6 +67,10 @@ define([
     fb,
```

Comment:
Visually not that much, but they have very different paths and sizes. Happy to condense to one, however I would prefer to do it outside this PR.